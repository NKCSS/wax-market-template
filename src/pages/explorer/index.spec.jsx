import { render } from '@testing-library/react'
import ExplorerPage from './index'

describe('::ExplorerPage', () => {
    it.skip('renders without crashing', () => {
        const { container } = render(<ExplorerPage tab="collections" />)
        expect(container).toMatchSnapshot()
    })
})
