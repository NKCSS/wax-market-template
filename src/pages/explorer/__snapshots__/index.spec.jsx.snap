// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`::ExplorerPage renders without crashing 1`] = `
<div>
  <div
    class="w-full h-auto min-h-full overflow-x-hidden m-auto ml-0 z-10 transition transform duration-200"
  >
    <ul
      class="border-tabs flex  h-12 my-10 rounded-md text-sm lg:text-base text-neutral border border-paper nav nav-tabs"
      role="tablist"
    >
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="collection-switch-tabpane-collections"
          aria-selected="true"
          class="nav-link active"
          data-rr-ui-event-key="collections"
          id="collection-switch-tab-collections"
          role="tab"
          type="button"
        >
          <div
            class="flex justify-center w-full h-full overflow-hidden rounded-md pl-5 text-white text-base md:text-lg text-center no-underline font-normal opacity-100 transition-opacity duration-500 hover:text-white hover:opacity-100 hover:underline selected:text-white selected:opacity-100 opacity-100 bg-primary text-secondary"
          >
            <div
              class="my-auto px-5"
            >
              Collections
            </div>
          </div>
        </button>
      </li>
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="collection-switch-tabpane-assets"
          class="nav-link"
          data-rr-ui-event-key="assets"
          id="collection-switch-tab-assets"
          role="tab"
          tabindex="-1"
          type="button"
        >
          <div
            class="flex justify-center w-full h-full overflow-hidden rounded-md pl-5 text-white text-base md:text-lg text-center no-underline font-normal opacity-100 transition-opacity duration-500 hover:text-white hover:opacity-100 hover:underline selected:text-white selected:opacity-100"
          >
            <div
              class="my-auto px-5"
            >
              Assets
            </div>
          </div>
        </button>
      </li>
    </ul>
    <div
      class="tab-content"
    >
      <div
        aria-labelledby="collection-switch-tab-collections"
        class="tab-pane active"
        id="collection-switch-tabpane-collections"
      >
        <div
          class="lg:justify-evenly px-6 min-h-60 grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-8"
        />
      </div>
    </div>
  </div>
</div>
`;
